webpack/assets/javascripts/react_app/components/ModelsTable/ModelsTable.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Table as PfTable } from 'patternfly-react';
+
+class ModelsTable extends React.Component {
+  componentWillMount() {
+    this.props.getModels(this.props.url);
+  }
+  render() {
+    const {
+      columns, filterFnc, results, store,
+    } = this.props;
+    return (
+      <PfTable.PfProvider
+        className="table-fixed"
+        striped
+        bordered
+        hover
+        columns={columns}
+        components={{
+          header: {
+            cell: props => (
+              <PfTable.Heading className={props.className} {...props.data}>
+                {props.children}
+              </PfTable.Heading>
+            ),
+          },
+          body: {
+            cell: props => (
+              <PfTable.Cell className={props.className} {...props.data}>
+                {props.children}
```

Comment:
missing table spinner while loading api results?

Reply:
there is also foreman's [Loader](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/common/Loader/index.js) - we should really consolidate all of these kind of components as its just a ui gaps (and bugs) waiting to happen between foreman and katello again....